work_queue/src/work_queue.c

```diff
@@ -84,6 +84,10 @@ See the file COPYING for details.
 
 #define MAX_NEW_WORKERS 10
 
+// Seconds for how often the user can be notified about when a task cannot fit to any workers
+// Used in check_task_fit_in_connected_workers()
+#define THRESHOLD 60000000
```

Comment:
let's make this one:
```
static timestamp_t  interval_tasks_fit_check  180000000;  //3 minutes in usecs
```

Reply:
Where should I place this variable? should this be a global variable?

Reply:
Yes, in the same place where wq_option_blocklist_slow_workers_timeout is. (Around line 125.)